standalone/pruntime/enclave/src/prpc_service.rs

```diff
@@ -379,32 +385,43 @@ pub fn init_runtime(
     ));
     drop(system_state);
 
-    let mut other_contracts: BTreeMap<ContractId, Box<dyn contracts::Contract + Send>> =
+    let mut contracts: BTreeMap<ContractId, Box<dyn contracts::Contract + Send>> =
         Default::default();
 
     if local_state.dev_mode {
         // Install contracts when running in dev_mode.
 
         macro_rules! install_contract {
             ($id: expr, $inner: expr) => {{
+                let contract_id = contract::id256($id);
                 let sender = MessageOrigin::native_contract($id);
                 let mq = send_mq.channel(sender, id_pair.clone());
-                let cmd_mq = PeelingReceiver::new_plain(recv_mq.subscribe_bound());
-                let evt_mq = PeelingReceiver::new_plain(recv_mq.subscribe_bound());
+                // TODO.kevin: use real contract key
+                let contract_key = local_ecdh_key.clone();
+                let cmd_mq = PeelingReceiver::new_secret(
```

Comment:
If we have two queues, is the order of the inputs preserved?

i.e. If on the blockchain we have [cmd1, ev1, cmd2], do we process them in order?

Reply:
> If we have two queues, is the order of the inputs preserved?

Yes, it is.

Reply:
Okay. That well fits our design